From 9187a520265723054fa1eec22af6dd113c4070b3 Mon Sep 17 00:00:00 2001
From: Jason Smith <jsmith@habana.ai>
Date: Sat, 24 May 2025 10:39:09 -0400
Subject: [PATCH] [SW-230673] pldm: reorder pdrs per spreadsheet

Some of the PDRs were added in a different order than how they are
defined in the spreadsheet.  Manually reorder them to reduce
differences when future commit auto-generates these files.

Change-Id: Ie1f9890a449dd50626ced5fadda64863b10b2125
Signed-off-by: Jason Smith <jsmith@habana.ai>
---

diff --git a/common/include/gaudi3/pldm_pdr_generated.h b/common/include/gaudi3/pldm_pdr_generated.h
index 858c5c6..d3dbfa1 100644
--- a/common/include/gaudi3/pldm_pdr_generated.h
+++ b/common/include/gaudi3/pldm_pdr_generated.h
@@ -13,21 +13,21 @@
 #include <pldm_state_set.h>
 
 enum pldm_aux_ridx {
-	PLDM_RIDX_EFFECTER_AUX_NAMES_ASIC_RESET,
-	PLDM_RIDX_SENSOR_AUX_NAMES_FW_STATE,
-	PLDM_RIDX_SENSOR_AUX_NAMES_ERROR_STATE_2,
-	PLDM_RIDX_SENSOR_AUX_NAMES_SECURITY_SPI_ERROR_STATE,
-	PLDM_RIDX_SENSOR_AUX_NAMES_SECURITY_KEY_STATE,
-	PLDM_RIDX_SENSOR_AUX_NAMES_SECURITY_STATE,
-	PLDM_RIDX_EFFECTER_AUX_NAMES_MARGIN_TOOL,
-	PLDM_RIDX_SENSOR_AUX_NAMES_ETH_SERDES_AVAILABILITY_OAM,
-	PLDM_RIDX_SENSOR_AUX_NAMES_ETH_SERDES_AVAILABILITY_RETIMER_1,
-	PLDM_RIDX_SENSOR_AUX_NAMES_ETH_SERDES_AVAILABILITY_RETIMER_2,
 	PLDM_RIDX_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_HIGH,
 	PLDM_RIDX_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_LOW,
 	PLDM_RIDX_SENSOR_AUX_NAMES_DEVICE_PDR_REPOSITORY_VERSION,
 	PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_LINK_WIDTH,
 	PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_LINK_SPEED,
+	PLDM_RIDX_SENSOR_AUX_NAMES_FW_STATE,
+	PLDM_RIDX_SENSOR_AUX_NAMES_SECURITY_KEY_STATE,
+	PLDM_RIDX_SENSOR_AUX_NAMES_SECURITY_STATE,
+	PLDM_RIDX_SENSOR_AUX_NAMES_ERROR_STATE_2,
+	PLDM_RIDX_SENSOR_AUX_NAMES_SECURITY_SPI_ERROR_STATE,
+	PLDM_RIDX_EFFECTER_AUX_NAMES_ASIC_RESET,
+	PLDM_RIDX_EFFECTER_AUX_NAMES_MARGIN_TOOL,
+	PLDM_RIDX_SENSOR_AUX_NAMES_ETH_SERDES_AVAILABILITY_OAM,
+	PLDM_RIDX_SENSOR_AUX_NAMES_ETH_SERDES_AVAILABILITY_RETIMER_1,
+	PLDM_RIDX_SENSOR_AUX_NAMES_ETH_SERDES_AVAILABILITY_RETIMER_2,
 	PLDM_RIDX_EFFECTER_AUX_NAMES_PCIE_MARGIN_TOOL_MODE_1,
 	PLDM_RIDX_EFFECTER_AUX_NAMES_PCIE_MARGIN_TOOL_MODE_2,
 	PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_DEVICE_ID,
diff --git a/common/src/gaudi3/ras/pldm/pldm_pdr_generated.c b/common/src/gaudi3/ras/pldm/pldm_pdr_generated.c
index cdfea96..ef1c074 100644
--- a/common/src/gaudi3/ras/pldm/pldm_pdr_generated.c
+++ b/common/src/gaudi3/ras/pldm/pldm_pdr_generated.c
@@ -360,24 +360,88 @@
 };
 
 struct pldm_pdr_name_record pldm_pdr_name_records[] = {
-	[PLDM_RIDX_EFFECTER_AUX_NAMES_ASIC_RESET] = { .aux_names = {
+	[PLDM_RIDX_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_HIGH] = { .aux_names = {
 		.header = {
-			.record_handle = PLDM_RH_EFFECTER_AUX_NAMES_ASIC_RESET,
+			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_HIGH,
 			.header_version = PLDM_PDR_HEADER_VERSION,
-			.type = PLDM_PDR_TYPE_EFFECTER_AUX_NAMES,
+			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
 			.record_change_number = 0,
 		},
 		.terminus_handle = 0,
-		.id = PLDM_SID_ASIC_RESET,
+		.id = PLDM_SID_ASIC_SERIAL_NUM_HIGH,
 		.count = 1,
-		.names = {
-			{
-				.name_string_count = 1,
-				.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
-				.name = asicReset_str,
-			},
-		},
+		.names = {{
+			.name_string_count = 1,
+			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
+			.name = asicSerialNumberHigh_str,
+		}},
 	}},
+	[PLDM_RIDX_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_LOW] = { .aux_names = {
+		.header = {
+			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_LOW,
+			.header_version = PLDM_PDR_HEADER_VERSION,
+			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
+			.record_change_number = 0,
+		},
+		.terminus_handle = 0,
+		.id = PLDM_SID_ASIC_SERIAL_NUM_LOW,
+		.count = 1,
+		.names = {{
+			.name_string_count = 1,
+			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
+			.name = asicSerialNumberLow_str,
+		}},
+	}},
+	[PLDM_RIDX_SENSOR_AUX_NAMES_DEVICE_PDR_REPOSITORY_VERSION] = { .aux_names = {
+		.header = {
+			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_DEVICE_PDR_REPOSITORY_VERSION,
+			.header_version = PLDM_PDR_HEADER_VERSION,
+			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
+			.record_change_number = 0,
+		},
+		.terminus_handle = 0,
+		.id = PLDM_SID_DEVICE_PDR_REPOSITORY_VERSION,
+		.count = 1,
+		.names = {{
+			.name_string_count = 1,
+			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
+			.name = devicePDRRepositoryVersion_str,
+		}},
+	}},
+	[PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_LINK_WIDTH] = { .aux_names = {
+		.header = {
+			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_PCIE_LINK_WIDTH,
+			.header_version = PLDM_PDR_HEADER_VERSION,
+			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
+			.record_change_number = 0,
+		},
+		.terminus_handle = 0,
+		.id = PLDM_SID_PCIE_LINK_WIDTH,
+		.count = 1,
+		.names = {{
+			.name_string_count = 1,
+			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
+			.name = currentPCIeLinkWidth_str,
+		}},
+	}},
+	[PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_LINK_SPEED] = { .aux_names = {
+		.header = {
+			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_PCIE_LINK_SPEED,
+			.header_version = PLDM_PDR_HEADER_VERSION,
+			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
+			.record_change_number = 0,
+		},
+		.terminus_handle = 0,
+		.id = PLDM_SID_PCIE_LINK_SPEED,
+		.count = 1,
+		.names = {{
+			.name_string_count = 1,
+			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
+			.name = currentPCIeLinkSpeed_str,
+		}},
+	}},
+
+
 	[PLDM_RIDX_SENSOR_AUX_NAMES_FW_STATE] = { .aux_names = {
 		.header = {
 			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_FW_STATE,
@@ -469,6 +533,24 @@
 				.name_string_count = 1,
 				.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
 				.name = signature_str,
+			},
+		},
+	}},
+	[PLDM_RIDX_EFFECTER_AUX_NAMES_ASIC_RESET] = { .aux_names = {
+		.header = {
+			.record_handle = PLDM_RH_EFFECTER_AUX_NAMES_ASIC_RESET,
+			.header_version = PLDM_PDR_HEADER_VERSION,
+			.type = PLDM_PDR_TYPE_EFFECTER_AUX_NAMES,
+			.record_change_number = 0,
+		},
+		.terminus_handle = 0,
+		.id = PLDM_SID_ASIC_RESET,
+		.count = 1,
+		.names = {
+			{
+				.name_string_count = 1,
+				.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
+				.name = asicReset_str,
 			},
 		},
 	}},
@@ -689,86 +771,6 @@
 				.name = Eth_Serdes_Availability_str,
 			},
 		},
-	}},
-	[PLDM_RIDX_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_HIGH] = { .aux_names = {
-		.header = {
-			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_HIGH,
-			.header_version = PLDM_PDR_HEADER_VERSION,
-			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
-			.record_change_number = 0,
-		},
-		.terminus_handle = 0,
-		.id = PLDM_SID_ASIC_SERIAL_NUM_HIGH,
-		.count = 1,
-		.names = {{
-			.name_string_count = 1,
-			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
-			.name = asicSerialNumberHigh_str,
-		}},
-	}},
-	[PLDM_RIDX_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_LOW] = { .aux_names = {
-		.header = {
-			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_ASIC_SERIAL_NUM_LOW,
-			.header_version = PLDM_PDR_HEADER_VERSION,
-			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
-			.record_change_number = 0,
-		},
-		.terminus_handle = 0,
-		.id = PLDM_SID_ASIC_SERIAL_NUM_LOW,
-		.count = 1,
-		.names = {{
-			.name_string_count = 1,
-			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
-			.name = asicSerialNumberLow_str,
-		}},
-	}},
-	[PLDM_RIDX_SENSOR_AUX_NAMES_DEVICE_PDR_REPOSITORY_VERSION] = { .aux_names = {
-		.header = {
-			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_DEVICE_PDR_REPOSITORY_VERSION,
-			.header_version = PLDM_PDR_HEADER_VERSION,
-			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
-			.record_change_number = 0,
-		},
-		.terminus_handle = 0,
-		.id = PLDM_SID_DEVICE_PDR_REPOSITORY_VERSION,
-		.count = 1,
-		.names = {{
-			.name_string_count = 1,
-			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
-			.name = devicePDRRepositoryVersion_str,
-		}},
-	}},
-	[PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_LINK_WIDTH] = { .aux_names = {
-		.header = {
-			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_PCIE_LINK_WIDTH,
-			.header_version = PLDM_PDR_HEADER_VERSION,
-			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
-			.record_change_number = 0,
-		},
-		.terminus_handle = 0,
-		.id = PLDM_SID_PCIE_LINK_WIDTH,
-		.count = 1,
-		.names = {{
-			.name_string_count = 1,
-			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
-			.name = currentPCIeLinkWidth_str,
-		}},
-	}},
-	[PLDM_RIDX_SENSOR_AUX_NAMES_PCIE_LINK_SPEED] = { .aux_names = {
-		.header = {
-			.record_handle = PLDM_RH_SENSOR_AUX_NAMES_PCIE_LINK_SPEED,
-			.header_version = PLDM_PDR_HEADER_VERSION,
-			.type = PLDM_PDR_TYPE_SENSOR_AUX_NAMES,
-			.record_change_number = 0,
-		},
-		.terminus_handle = 0,
-		.id = PLDM_SID_PCIE_LINK_SPEED,
-		.count = 1,
-		.names = {{
-			.name_string_count = 1,
-			.name_language_tag = PLDM_PDR_LANGUAGE_TAG_ENGLISH,
-			.name = currentPCIeLinkSpeed_str,
-		}},
 	}},
 	[PLDM_RIDX_EFFECTER_AUX_NAMES_PCIE_MARGIN_TOOL_MODE_1] = { .aux_names = {
 		.header = {
